rules/sinks/third_parties/sdk/stabilityai/python.yml

```diff
@@ -0,0 +1,13 @@
+
+#  Sink rule for ThirdParty SDK
+#  The id follows a format : "ThirdParties.SDK.<THIRD_PARTY_ORGANISATION>.<SUB_ORGANISATION_IF_APPLICABLE>"
+
+sinks:
+
+  - id: ThirdParties.SDK.StabilityAI
+    name: StabilityAI
+    domains:
+      - "https://platform.stability.ai/"
+    patterns:
+      - "(?i)(stability_sdk.interfaces.gooseai).*"
```

Comment:
Python methodFullname follows a different pattern you would see fullnames like `package.py/someFile/withinThePackage/Class.MethodName<args-and-return-type>` the usual dot notation may not match with python fullnames